examples/sequence-to-sequence/seq2seq_translator/seq2seq_dynet.py

```diff
@@ -0,0 +1,367 @@
+# Requirements
+
+from __future__ import unicode_literals, print_function, division
+import io
+import unicodedata
+import re
+import random
+import dynet as dy
+import time
+import math
+r = random.SystemRandom()
+
+# Data Preparation
+
+SOS_token = 0
+EOS_token = 1
+
+
+class Lang(object):
```

Comment:
Can you add a small docstring for each class/method/function? So that it is easier for people who want to reuse the code?